app/helpers/application_helper.rb

```diff
@@ -7,4 +7,14 @@ def signup_path?
 
     false
   end
+
+  def user_like_post?(one, two, three)
+    return true if Like.where(status: one).where(user_id: two).where(post_id: three).exists?
+
+    false
+  end
+
+  def unlike_id(one, two, three)
```

Comment:
- [ ] The same concept here also for no application logic in the helpers or in views.